studio/auth.py

```diff
@@ -68,7 +68,7 @@ def __init__(
                 self.email = email
                 self.password = password
             else:
-                self.email = input(
+                self.email = raw_input(
```

Comment:
I think this should be six.input() actually for python 2/3 compatibility

Reply:
on the second thought, this file is soon to go entirely when we switch to github authentication, so don't worry about it